src/app/services/theme/theme.service.ts

```diff
@@ -0,0 +1,213 @@
+import { DOCUMENT } from "@angular/common";
+import { Inject, Injectable } from "@angular/core";
+import * as Color from "color";
+
+/** List of palette options for website theming */
+export enum BawPalettes {
```

Comment:
```suggestion
export enum BawThemeColors {
```

Symmetry with https://getbootstrap.com/docs/5.0/utilities/colors/#map

I don't believe this should be a public type, in which name spacing and specificity are not required